boards/arm/s32z270dc2_r52/doc/index.rst

```diff
@@ -139,6 +139,13 @@ which supports CAN 2.0 and CAN FD protocol.
 CAN driver supports classic (CAN 2.0) and CAN FD mode. Remote transmission request is
 not supported as this feature is not available on NXP S32 CANXL HAL.
 
+CAN driver supports two modes of reception: RX Message Descriptor mode and RX FIFO mode.
+When using RX message descriptor, the RX filter can not match both CAN classic and
+CAN FD frames. Because driver needs to know the frame type to configure a RX filter
+match a CAN classic or a CAN FD frame. So it only supports CAN classic.
+When using RX FIFO, driver does not need to know this and it supports both
+CAN classic and CAN FD frames.
```

Comment:
In general, these type of comments do not belong in the board documentation, as they are specific to a driver, not a board.

Reply:
I removed this and this already in drivers